### DISCUSSION TITLE
TextInput should support relationship() like Select

### DISCUSSION THREAD

**Discussion by kotakomputer-ceo** [filamentphp/filament#10404] [Ideas]
**Forms\Components\TextInput:make('author_id')
    ->relationship(name: 'user', titleAttribute: 'name')**

Above code will error but working fine for Select. I hope you also add relationship() method for TextInput.
Currently we should use Placeholder with manual coding.

**Comment by luizsilva-dev** [comment]
the same problem here. I need use relationship with textInput.

**Comment by luizsilva-dev** [comment]
I found the solution.
Inside the schema you create a Group, call the relationship in the Group and place the TextInput inside the Group.

```
    ->schema([
        Group::make()
            ->relationship('table2')
            ->schema([
                TextInput::make('column_table2'),
            ]),
        Group::make()
            ->schema([
                TextInput::make('column_table1'),
            ]),
    ])
```

**Comment by john-aspire** [comment]
Hi, 

Although you can use the 'Group' to achieve this, it seems more logical to support the dot notation such as 'user.name' and the component picks it up automatically. This is how it works with the TextColumn on Table component: Tables\Columns\TextColumn::make('user.name')
